python/lsst/pipe/tasks/postprocess.py

```diff
@@ -767,10 +773,15 @@ class TransformObjectCatalogConnections(pipeBase.PipelineTaskConnections,
         doc="Per-Patch Object Table of columns transformed from the deepCoadd_obj table per the standard "
             "data model.",
         dimensions=("tract", "patch", "skymap"),
-        storageClass="DataFrame",
+        storageClass="ArrowAstropy",
         name="objectTable"
     )
 
+    def __init__(self, *, config=None):
+        super().__init__(config=config)
+        if config.multilevelOutput:
+            self.outputCatalog = dataclasses.replace(self.outputCatalog, storageClass="DataFrame")
```

Comment:
This is some next-level python stuff here.  So you can just do this to change one element of a dataclass?  And `self.outputCatalog.storageClass = "DataFrame"` doesn't work for some reason?